src/core/providers/strategies/LedgerProviderStrategy/helpers/updateLedgerAccounts.ts

```diff
@@ -0,0 +1,112 @@
+import { HWProvider } from '@multiversx/sdk-hw-provider/out';
+import BigNumber from 'bignumber.js';
+import { ProviderErrorsEnum } from 'types';
+import { fetchAccount } from 'utils/account/fetchAccount';
+import { ILedgerAccount } from '../types';
+import { LedgerStateManagerType } from '../types/ledgerProvider.types';
+
+type LedgerAccountsProps = {
+  manager: LedgerStateManagerType | null;
+  provider: HWProvider | null;
+};
+
+export const updateLedgerAccounts = async ({
```

Comment:
/*
  refresh the list of ledger addresses containing rows with:
  - checkbox 
  - address
  - balance
*/